charts/qfusion/v3.12.0/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: qfusion-installer
+  name: qfusion-installer
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app: qfusion-installer
+  name: installer-{{.Release.Namespace}}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Is there any reason to use `cluster-admin`, better to use `custom roles`.

Reply:
This must be use cluster-admin. Because operator will create servieAccount, clusterRole, clusterRolebinds when user create user in ower web.